### PR TITLE
PE-491 Run script to identify likely bank duplicates

### PR DESCRIPTION
**Pull request by ahmedlamidi**

### CODE REVIEW

File: brand/admin.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from django.contrib.admin.widgets import FilteredSelectMultiple
 from django.urls import reverse
 from django.utils.html import escape, format_html
+from django_object_actions import DjangoObjectActions
```
Comment (RogerTangos) [inline brand/admin.py:6]: django_object_actions needs to be added to `requirements.txt`

File: brand/admin.py [hunk 2]
```diff
@@ -269,9 +271,17 @@ class InstitutionCredentials(admin.ModelAdmin):
 
 
 @admin.register(Brand)
-class BrandAdmin(admin.ModelAdmin):
+class BrandAdmin(DjangoObjectActions,admin.ModelAdmin):
     form = CountriesWidgetOverrideForm
 
+    def Check_for_duplicates(self, request, query_set):
+        suggested_duplicates = return_all_duplicates()
+        return render(request,
```
Comment (RogerTangos) [inline brand/admin.py:279]: `render` is not imported

File: brand/admin.py (hunk 1 above)
Comment (RogerTangos) [inline brand/admin.py:6]: I'm glad the library is well supported but it seems a bit unnecedssary to use django_object_actions here. Is there a way to do this without introducing a new dependency?

**Review by RogerTangos — CHANGES_REQUESTED** [review]
I'm not seeing duplicates appear although the `suggested_duplicates` dictionary is populated

Screenshot 2023-10-10 at 11 40 00

**Comment by ahmedlamidi** [comment]
It should be working now, It was the html page that was not the same as the one running on my local system.

**Comment by RogerTangos** [comment]
On the HTML page,

- The original brand will also need to be linked to ensure they're not the same.
- It should just display the brand name.
- If there are duplicates it should only suggest one.

I think that a pass through the dictionary would fix the last two points.

Screenshot 2023-10-11 at 18 49 59

**Pull request closed without merging**